README.md

```diff
@@ -83,6 +83,20 @@ Update the admin user in the `database/seeders/DatabaseSeeder.php` file and run
 make art ARGS="db:seed"
 ```
 
+## Running the Application
+
+Run this command to start the app
+
+```bash
+make start
+```
+
+After completion, verify by visiting the endpoint below
+
+```bash
+http://localhost:8000/
```

Comment:
Here the port can be anything set in the APP_EXTERNAL_PORT env variable, make sure to update this to include this info.

Reply:
working on it.